blog/2023-05-04-announcing-realtime-reporting-for-stryker.mdx

```diff
@@ -1,39 +1,51 @@
 ---
 slug: announcing-realtime-reporting-for-stryker
-title: 'Announcing Realtime Reporting for Stryker'
+title: 'Announcing Real-Time Reporting for Stryker.NET'
```

Comment:
I have always referred to it as “realtime” and technically they are both correct ways of spelling it, thoughts?

Reply:
Other than that looks good 👍

Reply:
Yes, my thought was that it was advised by [Grammarly](https://app.grammarly.com/) to be better readable this way. Who am I to question Grammarly?

Reply:
However it should be Real-time and not Real-Time.

Reply:
I've updated it to "Real-time"